diff --git a/node_modules/@studiometa/playground/dist/front/js/components/Editor.js b/node_modules/@studiometa/playground/dist/front/js/components/Editor.js
index cb97ebc..7a7691e 100644
--- a/node_modules/@studiometa/playground/dist/front/js/components/Editor.js
+++ b/node_modules/@studiometa/playground/dist/front/js/components/Editor.js
@@ -36,7 +36,7 @@ class Editor extends Base {
       tabSize: 2,
       theme: themeIsDark() ? "vs-dark" : "vs"
     });
-    const disposeHTML = emmetHTML(monaco, ["html"]);
+    const disposeHTML = emmetHTML(monaco, ["html", "twig"]);
     const disposeCSS = emmetCSS(monaco, ["css"]);
     addJsAutocompletion(monaco.languages);
     this.$on(
diff --git a/node_modules/@studiometa/playground/dist/front/js/components/HtmlEditor.js b/node_modules/@studiometa/playground/dist/front/js/components/HtmlEditor.js
index fbf6b72..15f6fda 100644
--- a/node_modules/@studiometa/playground/dist/front/js/components/HtmlEditor.js
+++ b/node_modules/@studiometa/playground/dist/front/js/components/HtmlEditor.js
@@ -2,7 +2,7 @@ import { getHtml, setHtml } from "../store/index.js";
 import Editor from "./Editor.js";
 class HtmlEditor extends Editor {
   get language() {
-    return "html";
+    return "twig";
   }
   async getInitialValue() {
     return getHtml();
diff --git a/node_modules/@studiometa/playground/dist/front/js/components/Iframe.js b/node_modules/@studiometa/playground/dist/front/js/components/Iframe.js
index 9c768d7..c3f618b 100644
--- a/node_modules/@studiometa/playground/dist/front/js/components/Iframe.js
+++ b/node_modules/@studiometa/playground/dist/front/js/components/Iframe.js
@@ -112,7 +112,54 @@ class Iframe extends Base {
       tailwindScript.id = "tw";
       tailwindScript.addEventListener("load", () => {
         const tailwindConfig = this.doc.createElement("script");
-        tailwindConfig.textContent = "tailwind.config = { darkMode: 'class' };";
+        tailwindConfig.textContent = `tailwind.config = {
+          darkMode: 'class',
+          theme: {
+            screens: {
+              xxs: '0px',
+              xs: '480px',
+              s: '768px',
+              m: '1024px',
+              l: '1280px',
+              xl: '1440px',
+              xxl: '1920px',
+              xxxl: '2560px',
+            },
+            extend: {
+              zIndex: {
+                '-1': '-1',
+                1: '1',
+                2: '2',
+              },
+              transitionTimingFunction: {
+                'in-back': 'cubic-bezier(0.6, -0.28, 0.735, 0.045)',
+                'in-circ': 'cubic-bezier(0.6, 0.04, 0.98, 0.335)',
+                'in-cubic': 'cubic-bezier(0.55, 0.055, 0.675, 0.19)',
+                'in-expo': 'cubic-bezier(0.95, 0.05, 0.795, 0.035)',
+                'in-quad': 'cubic-bezier(0.55, 0.085, 0.68, 0.53)',
+                'in-quart': 'cubic-bezier(0.895, 0.03, 0.685, 0.22)',
+                'in-quint': 'cubic-bezier(0.755, 0.05, 0.855, 0.06)',
+                'in-sine': 'cubic-bezier(0.47, 0, 0.745, 0.715)',
+                'out-back': 'cubic-bezier(0.175, 0.885, 0.32, 1.275)',
+                'out-circ': 'cubic-bezier(0.075, 0.82, 0.165, 1)',
+                'out-cubic': 'cubic-bezier(0.215, 0.61, 0.355, 1)',
+                'out-expo': 'cubic-bezier(0.19, 1, 0.22, 1)',
+                'out-quad': 'cubic-bezier(0.25, 0.46, 0.45, 0.94)',
+                'out-quart': 'cubic-bezier(0.165, 0.84, 0.44, 1)',
+                'out-quint': 'cubic-bezier(0.23, 1, 0.32, 1)',
+                'out-sine': 'cubic-bezier(0.39, 0.575, 0.565, 1)',
+                'in-out-back': 'cubic-bezier(0.68, -0.55, 0.265, 1.55)',
+                'in-out-circ': 'cubic-bezier(0.785, 0.135, 0.15, 0.86)',
+                'in-out-cubic': 'cubic-bezier(0.645, 0.045, 0.355, 1)',
+                'in-out-expo': 'cubic-bezier(1, 0, 0, 1)',
+                'in-out-quad': 'cubic-bezier(0.455, 0.03, 0.515, 0.955)',
+                'in-out-quart': 'cubic-bezier(0.77, 0, 0.175, 1)',
+                'in-out-quint': 'cubic-bezier(0.86, 0, 0.07, 1)',
+                'in-out-sine': 'cubic-bezier(0.445, 0.05, 0.55, 0.95)',
+              },
+            }
+          }
+        };`;
         this.doc.head.append(tailwindConfig);
         resolve();
       });
